BUILD.md

```diff
@@ -181,7 +220,23 @@ To execute the script successfully you should have all the settings above, namel
  Still not clear why, but should be investigated.
  It might be connected to parallel execution, but likely not.
  
- 2. Any test which makes requests to commercetools platform environment may fail with:
+ ### Any test which makes requests to commercetools platform environment may fail with:
```

Comment:
if i know - i'd fix it already (((

As i remember, it could be token issue, handshake issue, concurrent modification, service unavailable and so on